<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentWidget</class>
 <widget class="QFrame" name="CommentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #1A1A1B;

border-radius: 18px;
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::StyledPanel</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="depthIconLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 17px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="postContentWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="authorLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="userAvatar">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 17px;
</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="username">
               <property name="styleSheet">
                <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 14px;
color:white;
margin-left:2px;
margin-top:3px;
margin-bottom:3px;</string>
               </property>
               <property name="text">
                <string>Gaudy3!p</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="usernamePoints">
               <property name="styleSheet">
                <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 500;
font-size: 10px;
line-height: 12px;
color:white;
margin-left:5px;
</string>
               </property>
               <property name="text">
                <string>190 points</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="authorLabel">
             <property name="styleSheet">
              <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 500;
font-size: 12px;
line-height: 14px;
color:white;
margin-bottom:11px;</string>
             </property>
             <property name="text">
              <string>1h ago</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="textBrowserContent">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 500 12pt &quot;Raleway&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="footerLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="likeButtonIcon">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="likeCount">
               <property name="styleSheet">
                <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 500;
font-size: 10px;
line-height: 12px;
color:white;
margin-left:5px;</string>
               </property>
               <property name="text">
                <string>103</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="commentButtonIcon">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="commentCount">
               <property name="styleSheet">
                <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 500;
font-size: 10px;
line-height: 12px;
color:white;
margin-left:5px;
margin-right:10px;</string>
               </property>
               <property name="text">
                <string>25</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="replyCommentButtonIcon">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="spacerLabel_Ignore">
               <property name="styleSheet">
                <string notr="true">font-family: 'Raleway';
font-style: normal;
font-weight: 500;
font-size: 10px;
line-height: 12px;
color:white;
margin-left:5px;
margin-right:10px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="divider">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgba(85, 85, 85, 0.7);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
